frontend/pages/DashboardPage/cards/ActivityFeed/GlobalActivityItem/GlobalActivityItem.tsx

```diff
@@ -266,8 +266,8 @@ const TAGGED_TEMPLATES = {
     if (activity.details?.mdm_platform === "microsoft") {
       return (
         <>
-          Mobile device management (MDM) was turned on for{" "}
-          <b>{activity.details?.host_display_name} (manual)</b>.
+          MDM features were turned on for{" "}
```

Comment:
Recommend removing the space at the end. I believe it will result in double spacing. Or the space can be removed from the start of the next line
```suggestion
          MDM features were turned on for
```